pageserver/src/metrics.rs

```diff
@@ -1489,6 +1489,15 @@ pub(crate) static LIVE_CONNECTIONS_COUNT: Lazy<IntGaugeVec> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+pub(crate) static COMPUTE_COMMANDS_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
+    register_int_counter_vec!(
+        "pageserver_compute_commands",
+        "Number of compute -> pageserver commands processed",
+        &["command"]
+    )
+    .expect("failed to define a metric")
+});
```

Comment:
Please
1. follow the pattern [like here](https://github.com/neondatabase/neon/blob/1a7565102a7689b9bffc836e9399ced3ff4dc983/pageserver/src/metrics.rs#L110-L146) to encapsulate the ~~shitty~~ brittle API of the prometheus crate behind nicer rust types. We do that for most new `..Vec` metrics nowadays so that a switch to a better metrics crate in the future is less invasive (Conrad is working on a new metrics crate)
2. After the step above, add this `Lazy` to the `preinitialize_metrics` `// custom` section [here](https://github.com/neondatabase/neon/blob/1a7565102a7689b9bffc836e9399ced3ff4dc983/pageserver/src/metrics.rs#L2994-L2997).

Reply:
Okay-dokay, changed that way.

It took me a while to understand what the pattern here is. There are many metrics that follow slightly different patterns, so it wasn't obvious what the right pattern is. I hope I got it right.